### DISCUSSION TITLE
WYSIWYG image captions

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#13393] [Archive]
Is it possible to add image captions within the wysiwyg editor?

I could not find a way to to chat.

What I mean is something like this:
![image](https://user-images.githubusercontent.com/10547444/169174738-34009310-96c0-41bd-b3a0-1340e0a8cd5c.png)

**Comment by rijkvanzanten** [comment]
I don't think TinyMCE has any support for `<figcaption>` 🤔
